src/fitnesseMain/FitNesseMain.java

```diff
@@ -49,6 +50,30 @@ public static void main(String[] args) {
   protected static void exit(int exitCode) {
     System.exit(exitCode);
   }
+  
+  // Testing code anaylsis tools
+  public String testMethod(){
+    
+  }
+  
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
+        return "badCommaWhiteSpaceBeforeAndAfter";
+    }
+
+  public void badGenericsWhiteSpace()
+  {List< String> strings = Arrays.asList("hello", "world");}
+
+  public boolean badOperatorWhiteSpace()
+  {
+      int sum = 1+2;
+      return sum==3;
+  }
+
+  @SafeVarargs public final void badAnnotations(String... badVarargsAnnotation)
+  {
+      @SuppressWarnings("default") Hello badSuppressWarningsAnnotation;
+  }
 
   public Integer launchFitNesse(Arguments arguments) throws Exception {
     ContextConfigurator contextConfigurator = ContextConfigurator.systemDefaults();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Line is longer than 80 characters (found 83). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'method def' child has incorrect indentation level 4, expected level should be 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck)